src/components/ProfilePage/LevelBox/LevelBox.tsx

```diff
@@ -1,25 +1,51 @@
-import { Skeleton } from '@mui/material';
-import { NoSelectTypography } from '../ProfilePage.style';
-import { TierBadgeBox } from './TierBox.style';
-
+import { Box } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import {
+  NoSelectTypography,
+  NoSelectTypographyTitle,
+} from '../ProfilePage.style';
+import InfoIcon from '@mui/icons-material/Info';
+import { Tooltip } from '@mui/material';
 interface LevelBoxProps {
   level?: number;
   loading: boolean;
 }
 
-export const LevelBox = ({ level, loading }: LevelBoxProps) => {
+export const LevelBox = ({ level }: LevelBoxProps) => {
+  const { t } = useTranslation();
   return (
-    <TierBadgeBox>
-      {loading ? (
-        <Skeleton
-          variant="text"
-          sx={{ fontSize: { xs: 18, sm: 22 }, minWidth: 60 }}
-        />
-      ) : (
-        <NoSelectTypography fontSize={{ xs: 12, sm: 18 }} fontWeight={700}>
-          {`LEVEL ${level}`}
-        </NoSelectTypography>
-      )}
-    </TierBadgeBox>
+    <Box>
+      <NoSelectTypography fontSize="14px" lineHeight="18px" fontWeight={700}>
+        LEVEL
+        <Tooltip
+          title={t('profile_page.levelInfo')}
+          placement="top"
+          enterTouchDelay={0}
+          arrow
+        >
+          <InfoIcon
```

Comment:
This is duplicated multiple times, would be good to refactorize (or create a styled component for it)